src/main/java/edu/ucla/library/iiif/auth/handlers/AccessTokenHandler.java

```diff
@@ -45,7 +46,7 @@
     /**
      * See {@link Config#ACCESS_TOKEN_EXPIRES_IN}.
      */
-    private final int myExpiresIn;
+    private final Optional<Integer> myExpiresIn;
```

Comment:
You were the one who convinced me the Optional should only be used on the outward facing API, not inner fields. :-)

Reply:
The alternative was to set it to something like `-1`, which felt wrong 🤷🏻

Reply:
Yes, that's the alt. Since that's what was done before Optionals, not sure it would be wrong. Since I've been on both sides of the Optionals for fields argument, I'm fine with whichever you choose. Just raising the issue...

Reply:
Since the use of Optional here is not causing issues like we were seeing with JPv3, I think it's the better approach